R/summary.QmethodRes.R

```diff
@@ -1,14 +1,14 @@
 summary.QmethodRes <- function(object, ...) {
```

Comment:
just FYI: I changed it from indices to names of objects, because I was afraid the indices would trip us up at some point, as `brief` and other objects are extended.